docs/package.json

```diff
@@ -5,7 +5,7 @@
     "build": "webpack",
     "build:clean": "rimraf build",
     "start": "webpack-dev-server --mode development --content-base docs/build/ --hot --progress --colors --host 0.0.0.0",
-    "start:local": "npm run start",
+    "start:prod": "npm run build && webpack-dev-server --mode production --content-base docs/build/ --hot --progress --colors --host 0.0.0.0",
```

Comment:
I'm glad we're adding this, for just this sort of debuggin'
It doesn't come up often, but when it does it's good to have this in the toolkit.